vnext/PropertySheets/Bundle.props

```diff
@@ -49,15 +48,15 @@
     <BundleEntryFile Condition="'$(BundleEntryFile)' == '' and Exists('$(BundleCommandWorkingDir)\index.windows.js')">index.windows.js</BundleEntryFile>
     <BundleEntryFile Condition="'$(BundleEntryFile)' == ''">index.js</BundleEntryFile>
 
-    <!-- Optional config file for the bundle command (metro will use metro.config.js by default) -->
```

Comment:
Is there a reason this is being removed?

Reply:
this was never hooked up to anything, nobody is actually using it.